cdc/redo/manager.go

```diff
@@ -118,14 +115,11 @@ type ManagerOptions struct {
 	ErrCh          chan<- error
 }
 
-type cacheRows struct {
-	tableID model.TableID
-	rows    []*model.RowChangedEvent
-	// When calling FlushLog for a table, we must ensure that all data of this
-	// table has been written to underlying writer. Since the EmitRowChangedEvents
-	// and FlushLog of the same table can't be executed concurrently, we can
-	// insert a simple barrier data into data stream to achieve this goal.
-	flushCallback chan struct{}
+type cacheEvents struct {
+	tableID    model.TableID
+	rows       []*model.RowChangedEvent
+	resolvedTs model.Ts
+	eventType  model.MqMessageType
```

Comment:
Type `model.MqMessageType` is mainly used for mq message, is there any other types that can differentiate RowChangedEvent from ResolvedTs event